pom.xml

```diff
@@ -46,14 +46,13 @@
         <lombok.version>1.18.12</lombok.version>
         <lombok-maven-plugin.version>1.18.10.0</lombok-maven-plugin.version>
         <delombok.output>target/generated-sources-for-javadocs</delombok.output>
-        <!-- TODO: Kotlin 1.4 - https://github.com/slackapi/java-slack-sdk/issues/545 -->
-        <kotlin.version>1.3.72</kotlin.version>
+        <kotlin.version>1.4.10</kotlin.version>
```

Comment:
Kotlin 1.3 and 1.4 are bin-compatible but just in case I held off upgrading this version until bumping a minor version.